pkg/scheduler/util.go

```diff
@@ -60,16 +58,39 @@ func loadSchedulerConf(confStr string) ([]framework.Action, []conf.Tier, error)
 		}
 	}
 
-	actionNames := strings.Split(schedulerConf.Actions, ",")
-	for _, actionName := range actionNames {
-		if action, found := framework.GetAction(strings.TrimSpace(actionName)); found {
-			actions = append(actions, action)
-		} else {
-			return nil, nil, fmt.Errorf("failed to found Action %s, ignore it", actionName)
+	for i, action := range schedulerConf.Actions {
+		if _, found := framework.GetAction(action.Name); !found {
+			return nil, fmt.Errorf("failed to found Action %s, ignore it", action.Name)
+		}
+
+		if action.Arguments == nil {
+			schedulerConf.Actions[i].Arguments = map[string]string{}
+		}
+
+		// set default value for backfill enabled
+		if _, found := schedulerConf.Actions[i].Arguments[conf.BackfillFlagName]; !found {
```

Comment:
It's not noly about the string, framework should not know anything about action.

Reply:
this check is already removed.

Reply:
this has been removed.